travel/res_config.py

```diff
@@ -36,6 +38,26 @@ def get_basic_passenger_limit(ir_config_parameter_pool, cr, uid, context=None):
         return limit
 
 
+def get_alert_address(ir_config_parameter_pool, cr, uid, context=None):
+    context = context or {}
```

Comment:
Hello @pedrobaeza 

Thank you for your coments. Can you elaborate on this context issue? I like very much the ```context = context or {}``` form and I've been using it for a couple of year now. Can you provide an example for your overwriting problem?